story-status-model-migration-run

From: Bryan Larsen <bryan@larsen.st>

Run your modified migration:

    $ rake db:migrate

That's it. The page to manage the story statuses should appear in the main navigation.

Now that we've got more structured statuses, let's do something with them...
---
 db/schema.rb |   11 +++++++++--
 1 files changed, 9 insertions(+), 2 deletions(-)

diff --git a/db/schema.rb b/db/schema.rb
index e9eaa8d..fb10fcd 100644
--- a/db/schema.rb
+++ b/db/schema.rb
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended to check this file into your version control system.
 
-ActiveRecord::Schema.define(:version => 20130109013414) do
+ActiveRecord::Schema.define(:version => 20130109022053) do
 
   create_table "projects", :force => true do |t|
     t.string   "name"
@@ -23,14 +23,21 @@ ActiveRecord::Schema.define(:version => 20130109013414) do
   create_table "stories", :force => true do |t|
     t.string   "title"
     t.text     "body"
-    t.string   "status"
     t.integer  "tasks_count", :default => 0, :null => false
     t.datetime "created_at"
     t.datetime "updated_at"
     t.integer  "project_id"
+    t.integer  "status_id"
   end
 
   add_index "stories", ["project_id"], :name => "index_stories_on_project_id"
+  add_index "stories", ["status_id"], :name => "index_stories_on_status_id"
+
+  create_table "story_statuses", :force => true do |t|
+    t.string   "name"
+    t.datetime "created_at"
+    t.datetime "updated_at"
+  end
 
   create_table "task_assignments", :force => true do |t|
     t.datetime "created_at"
